mindsdb/integrations/handlers/tripadvisor_handler/json_response_examples/location_details.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
What are these example files used for?

Reply:
Also should have documented this section. Each of the JSON files is the output of the API reference I am calling, so the developers would have an idea what's being received from the TripAdvisor side in JSON format.

Basically for documentation purposes.

Reply:
Should I delete them?

Reply:
Since we are returning the data in a table format, I do not think these will be required.

Reply:
Alright, will delete those files.